examples/evaluate_lightning_sequence_labeling.py

```diff
@@ -45,6 +48,13 @@ def run(
         output_path=root,
         evaluation_mode=evaluation_mode,
         tagging_scheme=tagging_scheme,
+        logging_kwargs={
+            "use_tensorboard": tensorboard,
+            "use_wandb": wandb,
```

Comment:
Could thing about making it more robust, and changing bool into str/enum and pass `wandb`, `mlflow` or others?

Reply:
Maybe it would be good to create wrapper for it that I suggest earlier such as `WandbLogger` and later we could use it as ``logging = WandbLogger(**logging_kwargs)``

Reply:
added `LightningLoggingConfig`